packages/eslint-config-react-app/package.json

```diff
@@ -28,7 +28,7 @@
     "eslint-plugin-jsx-a11y": "^6.3.1",
     "eslint-plugin-react": "^7.20.3",
     "eslint-plugin-react-hooks": "^4.0.8",
-    "eslint-plugin-testing-library": "^3.9.0"
+    "eslint-plugin-testing-library": "^3.9.0 || ^4.0.0"
```

Comment:
should 3.x really be a peerDep?

```suggestion
    "eslint-plugin-testing-library": "^4.0.0"
```

Reply:
This way we keep supporting older versions and this is not a breaking change